### PR TITLE
Sargis Vardanyan homework

### PR DESCRIPTION
**Pull request by SargisV**
Homework for 23.11.2021
Topic: Selection statements, if, else, switch, case.

### CODE REVIEW

File: homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask4.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+   if(c >= a){
+      max = c;
+   } else {
+      max = a;     
+   }
+ } else if(b >= a){
+     max = b;
+ } else {
+     max = a;
+ }
```
Comment (sirak92) [inline homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask4.java]: could you simplify it by assigning the first digit to max?

File: homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask5.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  if(a <= c){
+     min = a;
+  } else {
+    min = c;
+  }
+} else if(b <= c){
+   min = b;
+} else {
+  min = c;
+}
```
Comment (sirak92) [inline homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask5.java]: same here

**Review by sirak92 — COMMENTED** [review]
Good job

File: homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask4.java (hunk 1 above)
Comment (SargisV) [reply homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask4.java]: Maybe this way?        

int max = a;

if(max >= b) {
if(c >= max){
max = c;
  }
} else if(b >= c){
 max = b;
 } else {
 max = c;
 }

File: homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask5.java (hunk 2 above)
Comment (SargisV) [reply homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask5.java]: min = a;
if(min <= b) {
  if(c <= min){
   min = c;
   }
  } else if(b <= c){
   min = b;
  } else {
   min = c;
}

File: homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask4.java (hunk 1 above)
Comment (sirak92) [reply homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask4.java]: you don't need nested if statement
int max = a;
if(max > b) {
   max = b;
}
if (max > c) {
    max = c;
}

File: homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask5.java (hunk 2 above)
Comment (sirak92) [reply homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask5.java]: see the above comment

File: homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask4.java (hunk 1 above)
Comment (SargisV) [reply homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask4.java]: Okay thanks

**Pull request merged**